tensorflow/python/ops/image_ops_impl.py

```diff
@@ -3681,17 +3681,24 @@ def extract_glimpse(
     
   Usage Example:
     ```python
-    BATCH_SIZE = 2
-    IMAGE_HEIGHT = 256
-    IMAGE_WIDTH = 256
-    CHANNELS = 3
-    GLIMPSE_SIZE = (24, 24)
-    image = tf.random.normal(shape=(BATCH_SIZE, IMAGE_HEIGHT, 
-    IMAGE_WIDTH, CHANNELS) )
-    offsets = tf.random.uniform(shape=(BATCH_SIZE, 2))
+    BATCH_SIZE = 1
+    IMAGE_HEIGHT = 5
+    IMAGE_WIDTH = 5
+    CHANNELS = 1
+    GLIMPSE_SIZE = (3, 3)
+    image = tf.random.uniform(shape=(BATCH_SIZE, IMAGE_HEIGHT, 
+      IMAGE_WIDTH, CHANNELS) )
     output = tf.image.extract_glimpse(image, size=GLIMPSE_SIZE, 
-    offsets=offsets)
-    print(output.shape)
+      offsets=[[1, 1]], centered=False, normalized=False)
+    print(image)
+        <tf.Tensor: id=172, shape=(3, 3), dtype=float32, numpy=
+        array([[0.98461604, 0.14595735, 0.3296578 ],
```

Comment:
Showing particular random numbers is not super helpful. Can you put numbers like 1. 2. 3. etc in the image so a reader has an idea of where each input ended up?

Reply:
Hey I've made the changes, let me know if it's fine or do we need more changes?